### ISSUE TITLE
Question: standard deviation on cifar10

### ISSUE THREAD

**Issue by Sirius083** [DeepInsight-PCALab/MixNet#5]
In cifar10.py Line 105
transforms.Normalize((0.4914, 0.4822, 0.4465), (0.2023, 0.1994, 0.2010))
The standard deviation on cifar10 is [0.24703223 0.24348513 0.26158784]
and I search online
How do you calculate the previous standard deviation?
Thanks in advance

**Comment by whai362** [comment]
There is a standard implementation of CIFAR dataloader on torchvision.datasets.

**Comment by Sirius083** [comment]
Thanks, can you tell me the standard deviation of cifar100 under your implementation? I am using tensorflow to try you code.
Thanks

**Comment by Sirius083** [comment]
https://gist.github.com/weiaicunzai/e623931921efefd4c331622c344d8151
Sorry to bother, I find the answer here
Thanks
